exercises/practice/complex-numbers/complex-numbers.lua

```diff
@@ -0,0 +1,13 @@
+local Complex
+
+Complex = function(r, i)
+  local c = {
+    --
+  }
+
+  return setmetatable(c, {
+    --
+  })
+end
+
+return Complex
```

Comment:
I tried to include a few breadcrumbs here to help lead people in the right direction

Reply:
You could add [a document](https://exercism.org/docs/building/tracks/practice-exercises#h-documentation-files) (maybe instructions.append.md) that links to the "Closure based objects" part of https://web.archive.org/web/20240104094707/http://lua-users.org/wiki/ObjectOrientationTutorial

Reply:
Sorry about the delay. I should be able to do this over the weekend.

Reply:
Thanks for the suggestion. I added an append to introduce closure-based OO and operator overloading.